src/plugins/springroll-plugin.js

```diff
@@ -1,20 +1,29 @@
 import Phaser from 'phaser';
-import { Application, CaptionPlayer, HtmlRenderer, Debugger } from 'springroll';
+import { Application, CaptionPlayer, HtmlRenderer, Debugger, IHintPlayer, HintSequencePlayer } from 'springroll';
+
+class TestPlayer extends IHintPlayer{
+    play()
+    {
```

Comment:
Is there anything more we can do with this? doesn't seem like a very good demo is all.

Reply:
that wasn't supposed to be in here, that was me doing science.
it's been removed.